src/main.cu

```diff
@@ -337,7 +335,7 @@ int main(int argc, const char** argv) {
         std::ofstream csv_step(csvFileNameStep);
 
         if (experiment.histogram) {
-            csv << "repetition,grid_width,pop_size,p_occupation,resolution_iterations,mean_unresolved_count" << std::endl;
+            csv << "repetition,grid_width,pop_size,p_occupation,mean_pop_count,resolution_iterations,mean_unresolved_count" << std::endl;
```

Comment:
is it population count when the population (of agents) never changes?

Reply:
Yes as it is the mean over the number of steps of the simulation. It is the sugarscape agents rather than environment agents and it changes over time.

Reply:
There is only one type of agent in main.cu, which does not change over time, there is no agent creation or destruction in this model.

Reply:
The pop count is derived from the unresolved agents. Its an odd abstraction as the "agents" are the sugarscape agents. E.g. Cells which are occupied. The cells are the actual FLAME GPU agents which have a fixed pop size.